styles.css

```diff
@@ -1,36 +1,856 @@
+  background-image: url('./images/mobile_intro_bg.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 2rem 2.5rem;
+  padding-top: 5rem;
+}
+
+.hero-greeting {
+  color: #ec5242;
+  text-align: left;
+  font-family: var(--font-lato);
+  font-size: 2rem;
+  font-weight: 500;
+}
+
+.hero-title {
+  color: #ec5242;
+  background: url(./images/flame.jpg);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  font-size: 4.5rem;
+  font-family: var(--font-lato);
+  font-weight: 900;
+  line-height: 4.5rem;
+  width: 100%;
+  margin-top: 1.3rem;
+  margin-bottom: 1.8rem;
+}
+
+.hero-descr {
+  font-family: var(--font-lato);
+  font-size: 1.7rem;
+  line-height: 3rem;
+  font-weight: 700;
+  color: var(--color-gray-text);
+  border: 4px solid #ffff;
+  background-color: #f7f7f7;
+  width: 100%;
+  padding: 2rem;
+}
+
+.hero-date {
+  font-family: var(--font-cocogoose);
+  color: var(--color-primary);
+  font-size: 2.6rem;
+  margin-top: 1.8rem;
+}
+
+.hero-location {
+  color: var(--color-primary);
+  font-family: var(--font-lato);
+  font-size: 1.68em;
+  padding-bottom: 2rem;
+}
+
+@media (min-width: 768px) {
+  .hero-container {
+    background-image: url('./images/main_big.png');
+    background-repeat: no-repeat;
+    background-size: cover;
+    padding: 60px 10%;
+  }
+
+  .hero-greeting {
+    font-size: 3.5rem;
+    font-weight: 400;
+  }
+
+  .hero-title {
+    margin-top: 1rem;
+    font-size: 7.5rem;
+    line-height: 8rem;
+  }
+
+  .hero-descr {
+    margin-top: 3rem;
+    width: 80%;
+    font-size: 1.8rem;
+    line-height: 2.5rem;
+  }
+
+  .hero-date {
+    font-size: 2.7rem;
+  }
+}
+
+/* Program Section */
+.main-program {
+  font-family: var(--font-lato);
+  color: var(--color-white);
+  font-size: 2rem;
+  background-image: url('./images/pattern_bg.png');
+  background-repeat: repeat;
+  background-color: #272930;
+  padding-top: 2rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+.orange-line {
+  width: 6rem;
+  border-bottom: 1px solid#ec5242;
+  margin: 0 auto;
+}
+
+.program-title {
+  gap: 1em;
+  padding: 2em;
+  flex-direction: column;
+}
+
+.flex-container {
+  display: flex;
+  align-items: center;
+}
+
+.main-program-title {
+  text-align: center;
+  color: white;
+}
+
+.program-categories {
+  display: grid;
+  gap: 1em;
+  justify-items: center;
+}
+
+.program-categories-details span {
+  font-size: 1.4em;
+  color: #ec5242;
+  text-align: start;
+  width: 10rem;
+}
+
+.program-categories-details p {
+  text-align: start;
+  line-height: 1.5em;
+  flex: 2 0 0;
+  color: var(--color-white);
+  font-size: 1.6rem;
+}
+
+.program-categories-details {
+  gap: 1em;
+  min-width: 100%;
+  padding: 0.75em;
+  font-size: 0.75em;
+  background-color: #f7f7f71a;
+  color: var(--color-white);
+}
+
+.btn-program {
+  font-family: var(--font-lato);
+  background-color: var(--color-accent);
+  color: var(--color-white);
+  font-size: 01em;
+  width: 70%;
+  height: 4em;
```

Comment:
- [ ] Hey @peteryeungtt 👋 you have done an amazing job so far but there are some little things you have to fix so that you can move forward. I would suggest that you increase the width of the `.btn-program` selector to `85%` so that the button looks exactly like the Original design.